docs/faq.rst

```diff
@@ -130,3 +130,28 @@ How do I shutdown a KSQL environment?
    .. code:: bash
 
        $ confluent destroy
+
+========================================================
+Can KQL connect to a Apache Kafka cluster over SASL_SSL?
+========================================================
+
+Yes! Internally, KSQL uses standard Kafka Consumers and Producers, which you
+can configure to connect to a secure Kafka cluster as you would for any app.
+
+For example, adding the following entries, to the property file you use to start
+KSQL, will enable KSQL to connect to a cluster secured using _PLAIN_ SASL,
```

Comment:
"to connect to a Kafka cluster" -- just want to be extra-clear because we also say "KSQL cluster".

Reply:
done